portal/src/components/ContentLink/style.scss

```diff
@@ -3,7 +3,7 @@
 .jkl-portal__skip-to-content {
     z-index: $z-index--floating;
     padding: $component-spacing--medium;
-    background-color: $hvit;
+    background-color: transparent;
```

Comment:
er ikke transparent default? 🤔

Reply:
Ser ikke slik ut nei, blir hvit bak

Reply:
Hmm.. Jeg tror jeg satte denne hvit med vilje. Denne legger seg jo bare øverst til venstre på skjermen, så den kan overlappe ting som logo, etc. Synes derfor den bør ha bakgrunnsfarge. Kan evt. bruke `inherit` for å arve bakgrunnsfarge fra siden

Reply:
Tja, men det er jo ikke en delt komponent så jeg vet ikke om det har noe å bety i dette tilfelle

Reply:
Ser at den ikke overlapper her nei, så da bør det gå fint som det er 😊